tests/foreman/ui/test_setting.py

```diff
@@ -952,3 +949,143 @@ def test_positive_update_token_duration_param(self):
                     self.saved_element = self.settings.get_saved_value(
                         self.tab_locator, self.param_name)
                     self.assertEqual(param_value, self.saved_element)
+
+    @stubbed
+    @tier1
+    def test_negative_non_admin_access(self):
```

Comment:
agreed, if the access is being testes to "settings" page, so it must go in test_settings, otherwise test_user/test_role will center too much RBAC tests, if changes are needed in future will be in a separate place.. I agree to keep it in settings